doc/Running_Motr_Across_a_Cluster.md

```diff
@@ -101,9 +100,9 @@ This document provides information on how to build motr from source and then run
 	                 #profiles:
```

Comment:
Please check tabs setting here same for line # 101, # 102, # 200, # 201, # 202

Reply:
yes will check and fix it.